emulators/krunkit/files/patch-makefile.diff

```diff
@@ -0,0 +1,23 @@
+--- Makefile	2024-11-14 23:22:31
++++ Makefile.new	2024-12-27 01:03:33
+@@ -1,9 +1,9 @@
+ OS = $(shell uname -s)
+-KRUNKIT_RELEASE = target/release/krunkit
+-KRUNKIT_DEBUG = target/debug/krunkit
++KRUNKIT_RELEASE = target/$(CARGO_BUILD_TARGET)/release/krunkit
++KRUNKIT_DEBUG = target/$(CARGO_BUILD_TARGET)/debug/krunkit
+ 
+ ifeq ($(PREFIX),)
+-    PREFIX := /usr/local
++    PREFIX := @@DEFAULT_PREFIX@@
+ endif
```

Comment:
Same nitpick here: this could potentially be done on the command line to avoid the reinplace.

Reply:
Ditto (for reference: https://github.com/macports/macports-ports/pull/27208#discussion_r1921004594)